### PR TITLE
Merging the AWS Resource Tags

### PR DESCRIPTION
**Pull request by sairameshv**
This change is done in accordance with the following enhancement proposal.
EP: https://github.com/openshift/enhancements/blob/master/enhancements%2Fapi-review%2Fcustom-tags-aws.md
Latest EP's PR: https://github.com/openshift/enhancements/pull/1051

/hold until the above PR is merged

### CODE REVIEW

**Comment by sairameshv** [comment]
cc: @TrilokGeer

**Comment by sairameshv** [comment]
cc: @bharath-b-rh

File: pkg/operator/starter.go [hunk 1]
```diff
@@ -310,6 +332,29 @@ func withCustomTags(infraLister v1.InfrastructureLister) deploymentcontroller.De
 			}
 			container.Args = append(container.Args, tagsArgument)
 		}
+		recorder.Eventf("Successfully updated the user tags", "Tag Pairs: %s", tagPairs)
```
Comment (jsafrane) [inline pkg/operator/starter.go:335]: This function will be called with every `ClusterCSIDriver` CR update, which can be quite often during cluster installation. It should emit an event if anything really changes, not on every successful sync.

Comment (sairameshv) [reply pkg/operator/starter.go:335]: Agreed.

**Pull request closed without merging**